changelog/53120.fixed.md

```diff
@@ -0,0 +1 @@
+Masquerade property will not default to false turning off masquerade if not specified.
```

Comment:
Seems the team does not have issue with this. Can we just change the file extension here to `changed` so we can emphasize to users that this is a change they will need to be aware of.

```suggestion
Masquerade property will now default to false turning off masquerade if not specified.
```

Reply:
Probably will be better something like this ```

Masquerade property will now default to not specified. Previously it was defaulting to false leading to turning off masquerade if not specified.
```

What is suggested is exactly what is fixed.

Reply:
Oh doh! your right, can we still get this change tho: Can we just change the file extension here to `changed` instead of `fixed`. so we can emphasize to users that this is a change they will need to be aware of.

Reply:
Done.